frontend/src/components/GroupBar/index.jsx

```diff
@@ -13,26 +15,80 @@ function GroupBar( {id, changeId, page} ) {
     }, []);
 
     const changeToEditMode = () => {
+        if(editMode){
+            setEditGroup({id:-1,name:"",description:""})
+            setGroups(groups.filter((element)=> element.id !== -1))
+        }
         setEditMode(!editMode);
     }
 
-    const changeInformation = () => {
-        
+    const setupCreateGroup = () =>{
+        if(!groups.some((element)=> element.id == -1))
+            setGroups([...groups,{id:-1,name:"",description:""}])
     }
 
+    const changeEditGroupName = (name) =>{
```

Comment:
Change `changeEditGroupName` to `editGroupName`

Reply:
Ele da Change do nome na variável editgroup